pkgs/top-level/static.nix

```
@@ -162,6 +162,10 @@ in {
     # --disable-shared flag
     stdenv = super.stdenv;
   };
+  arrow-cpp = super.arrow-cpp.override {
+    static = true;
```

Need to fix an eval error:
```suggestion
    enableShared = true;
```